Realm/ObjectServerTests/RLMCollectionSyncTests.mm

```diff
@@ -76,7 +77,9 @@ - (void)roundTripWithPropertyGetter:(RLMSet *(^)(id))propertyGetter
             RLMResults<RLMSetSyncObject *> *results
                 = [RLMSetSyncObject allObjectsInRealm:realm];
             if (RLMSetSyncObject *obj = results.firstObject) {
-                if (propertyGetter(obj).count == 0) {
+                CHECK_COUNT(1, RLMSetSyncObject, realm);
+                if (propertyGetter(obj).count == 0
+                    && otherPropertyGetter(obj).count == 0) {
```

Comment:
Wrapping the line here makes it look like this line is part of the block following the if rather than part of the condition.